File: lec_04/loop_sum.cpp
#include <iostream>

using namespace std;

int loop_sum(int number) {
    int arr[number];
    int sum = 0;
    for (int i = 0; i < number; i++)
        cin >> arr[i];
    for (int i = 0; i < number; i++)
        sum += arr[i];
    return sum;
}


int main() {
    int number;
    cin >> number;
    cout << loop_sum(number) << '\n';
    return 0;
}

File: lec_04/loop_sum.py
def loop_sum(array):
    sum = 0
    for i in array:
        sum += i
    return sum


list = []
print('The number of elements in list:')
i = int(input())
print('Append data in list of:')
for i in range(i):
    data = int(input())
    list.append(data)
print("Sum of array = %s" %loop_sum(list))